docs/en/manuals/physics-joints.md

```diff
@@ -12,7 +12,7 @@ Defold supports joints for 2D physics. A joint connects two collision objects us
 * **Weld (physics.JOINT_TYPE_WELD)** - A weld joint attempts to constrain all relative movement between two collision objects. The weld joint can be made soft like a spring with a frequency and damping ratio. In Box2D referred to as a Weld joint.
 * **Spring (physics.JOINT_TYPE_SPRING)** - A spring joint keeps two collision objects at a constant distance from each other. The spring joint can be made soft like a spring with a frequency and damping ratio. In Box2D referred to as a Distance joint.
 * **Slider (physics.JOINT_TYPE_SLIDER)** - A slider joint allows for relative translation of two collision objects along a specified axis and prevents relative rotation. In Box2D referred to as a Prismatic joint.
-* **Wheel (physics.JOINT_TYPE_WHEEL)** - A wheel joint restricts a point on bodyB to a line on bodyA. The wheel joint also provides a suspension spring. In Box2D referred to as a Weld joint.
+* **Wheel (physics.JOINT_TYPE_WHEEL)** - A wheel joint restricts a point on bodyB to a line on bodyA. The wheel joint also provides a suspension spring. In Box2D referred to as a wheel joint.
```

Comment:
Could you please change to Wheel to match the other joint descriptions. A bonus also if you could add links to Box2d joint types for the Wheel joint and the other joints in that list!

```md
In Box2D referred to as a [Wheel joint](https://box2d.org/documentation/group__wheel__joint.html#details)
```

Reply:
Updated, made links to box2d documentation for most joints except for "rope" as it no longer exists in box2d documentation.